<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HuffmanMainClass</class>
 <widget class="QMainWindow" name="HuffmanMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Huffman</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Desktop/新建文件夹/resource/树.png</normaloff>../../../../../Desktop/新建文件夹/resource/树.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#HuffmanMain{background-image: url(:/background.png);}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>28</y>
      <width>177</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 28pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;HUFFMAN&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>421</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 16pt &quot;Microsoft YaHei UI Light&quot;;
background-color:rgba(255,255,255,0.1);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="codeTab">
     <property name="font">
      <font>
       <family>Microsoft YaHei UI Light</family>
       <pointsize>16</pointsize>
       <weight>3</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>CODE</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>259</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 16pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Please input your words or upload:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>33</y>
        <width>394</width>
        <height>111</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>141</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft JhengHei UI Light'; font-size:10pt; font-weight:24; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>If there is, please input relative coding or upload:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="encodeHfmTextV">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>394</width>
        <height>118</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16000000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI Light</family>
        <pointsize>10</pointsize>
        <weight>3</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="textUploadButton">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>5</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="treeUpload_en">
      <property name="geometry">
       <rect>
        <x>376</x>
        <y>150</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="decodeTab">
     <attribute name="title">
      <string>DECODE</string>
     </attribute>
     <widget class="QTextEdit" name="binaryTxtView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>33</y>
        <width>394</width>
        <height>111</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>141</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 12pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>Please input relative coding or upload:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 16pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Please input your binary string or upload:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="decodeHfmTextV">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>394</width>
        <height>118</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>111111</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI Light</family>
        <pointsize>10</pointsize>
        <weight>3</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 10pt &quot;Microsoft JhengHei UI Light&quot;;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="binaryUploadButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>5</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="treeUpload_de">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</normaloff>../../../../../Desktop/新建文件夹/resource/upload.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>479</y>
      <width>181</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 26pt &quot;Microsoft YaHei UI Light&quot;;</string>
    </property>
    <property name="text">
     <string>DISPLAY 〉</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="HuffmanMain.qrc"/>
  <include location="../../../../../Desktop/新建文件夹/Test/Test/back.qrc"/>
  <include location="back.qrc"/>
 </resources>
 <connections/>
</ui>
